### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marnovo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
 and mapped into a regular 54x54 = 2916 grid and described the latter in the [blog post](https://blog.sourced.tech/post/lapjv/).
 Thus every repository appears in a single specific cell of that grid.
 
-Given the identity matching, we had the contributions history and could even roughly estimate the impact as the ratio of
-contributed commits to the whole number of commits in the main branch of the project. We color the corresponding
-cell for each weighted contribution by the chosen user and compose the profile image. The complete pipeline
-is in the [legacy notebook](legacy/from_scratch.ipynb) which can not be directly executed without the datasets and
-the last step which draws the images is fully functional and you can [play with it](profile.ipynb).
-
-For example, this is
```
Comment (vmarkovtsev) [inline README.md]: This is a clear mismerge - I added the links to notebooks

File: README.md [hunk 2]
```diff
@@ -1,38 +1,31 @@
 # Developer Similarity
 
-This is a legacy from source{d}'s research in developer similarity once it was a [recruitment business](http://web.archive.org/web/20161020170426/http://sourced.tech/).
+This is a legacy from [source{d}](http://www.sourced.tech/)'s research in developer similarity when its main goal was to [match developers and companies](http://web.archive.org/web/20161020170426/http://sourced.tech/).
```
Comment (vmarkovtsev) [inline README.md:3]: how about "link developers with companies"? there are many "match"-es below

File: README.md [hunk 3]
```diff
@@ -1,38 +1,31 @@
...
 
-source{d} had (and is going to have again by the end of 2017) a nearly complete mirror of GitHub repositories.
-Everything related to working with developer metadata requires understanding which email addresses corerespond
-to which GitHub users. GitHub ToU directly forbids to use API for recruitment purposes which we clearly had,
-so we saw no easy way to obtain that information. Luckily, we were able to perform identity matching using
-various heuristics for Git commit signatures. For example, given two different signatures with the same name
-in the same repository, it is likely that the different email addresses belong to the same user. We never
-measured the accuracy of our heuristics - back then we were a humble startup - but our CTO claims that
-it worked reasonably well in practice.
+source{d} runs a massive-scale data retrieval pipeline that among other sources nearly mirrors all GitHub repositories.
```
Comment (vmarkovtsev) [inline README.md:7]: which at ~the~ **that** time? Is this a misprint or a feature? :)

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (marnovo) [reply README.md:3]: Let's settle for connect, then.

File: README.md (hunk 3 above)
Comment (marnovo) [reply README.md:7]: Feature. Both work.

File: README.md (hunk 1 above)
Comment (marnovo) [reply README.md]: Fixed.

**Pull request merged**